datafusion/core/src/physical_optimizer/enforce_distribution.rs

```diff
@@ -1738,8 +1738,11 @@ mod tests {
             _t: DisplayFormatType,
             f: &mut std::fmt::Formatter,
         ) -> std::fmt::Result {
-            let expr: Vec<String> = self.expr.iter().map(|e| e.to_string()).collect();
-            write!(f, "SortRequiredExec: [{}]", expr.join(","))
+            write!(
+                f,
+                "SortRequiredExec: [{}]",
+                PhysicalSortExpr::format_list(&self.expr)
```

Comment:
this is a cleanup to use the same code everywhere to format `[PhysicalSortExpr]`